.storybook/preview.tsx

```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import { ThemeProvider } from "metabase/ui";
 
-import "metabase/css/core/index.css";
-import "metabase/css/vendor.css";
-import "metabase/css/index.module.css";
-import "metabase/lib/dayjs";
+const isEmbeddingSDK = process.env.IS_EMBEDDING_SDK === "true";
+
+if (!isEmbeddingSDK) {
```

Comment:
for the sake of testing, we should be on a blank page with no css, otherwise we can't test if the provider is actually setting the css